drakcore/debian/rules

```diff
@@ -16,4 +16,4 @@ build:
 	chmod +x drakcore/systemd/minio
 
 override_dh_virtualenv:
-	dh_virtualenv --python python3
+	dh_virtualenv --python python3.7
```

Comment:
This (on the other hand) would break Ubuntu 20.04 build, because it has Python 3.8 :thinking:

Closing for now. I will test the latest release on Xubuntu to check whether we can force it to use Python >= 3.7